src/common/Button/Button.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ onClick, children, style, className, ...props }) => {
+const Button = ({ onClick, children, type, style, className, ...props }) => {
```

Comment:
With the way you've created this component: `type`, `className`, `style`, `onClick` properties should all be available within the `props` object if not specified directly. I.e., if you didn't make this change, you could still access `type` via `props.type`. Moreover, you may pass the type even without the changes since you're passing `{...props}` to the button component